test/servico/cst10Test.java

```diff
@@ -84,7 +79,8 @@ public void tearDown() {
         
         notificaçãoService.aceitarSubstituicao(ausencias.get(0).id);
         
-        ausencias = notificaçãoService.listarAusenciasPorSubstituto("Professor3");
+        ausencias = 

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['method def' child have incorrect indentation level 8, expected level should be 4.](https://www.codacy.com/app/calebepb/substituicao/file/2777960760/issues/source?bid=3364462&fileBranchId=3364484#l82)